packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -200,7 +200,7 @@ export const Tooltip = ({
     <>
       <ContainerElement
         ref={elementRef}
-        className={targetWrapperClassName}
+        className={cn(styles.TooltipContainer, targetWrapperClassName)}
```

Comment:
question or a possible nitpick - do we use camelCase or CapitalCamelCase in f36?

Reply:
With regards to CSS we seem to use a mix, of course 🙃😂 Almost all of it is like this though. I've always known it as PascalCase 🙂

Reply:
🙃
I'd guess we could fix it then in the whole file so it's consistent at least on the 'per file' basis?

Reply:
I think the PascalCase one is a class from the css file, not sure why we are using PascalCase to name the classes but I think every CSS class is using that pattern

And `targetWrapperClassName` is a prop and we are always using camelCase for props (this one makes sense for me hahahaha)

One good news is that we will move all styles to emotion so we could make everything camelCase then 🙌

Reply:
Yeah, CSS file class names are PascalCase, everything else is camelCase. However, I've seen camelCase names from CSS in a few files as well 😅